Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -521,3 +521,33 @@ def test_get_remote_data_no_attachment(mocker):
     res = get_remote_data_command(client, args, params)
     assert res[1]['Contents'] == 'This is a comment'
     assert len(res) == 2
+
+
+def test_get_remote_data_no_entries(mocker):
+    """
+    Given:
+        -  ServiceNow client
+        -  arguments: id and LastUpdate(set to lower then the modification time).
+        -  ServiceNow ticket
```

Comment:
please add here that the file and comments returned are ones that were sent from xsoar